pennylane/qchem/openfermion_obs.py

```diff
@@ -335,40 +336,28 @@ def two_particle(matrix_elements, core=None, active=None, cutoff=1.0e-12):
     ...                               [ 0.00000000e+00,  7.05105632e-01]]]])
     >>> v_op = two_particle(matrix_elements)
     >>> print(v_op)
-    0.3411947665760211 [0^ 0^ 0 0]
-    + 0.08950028803070323 [0^ 0^ 2 2]
-    + 0.3411947665760211 [0^ 1^ 1 0]
-    + 0.08950028803070323 [0^ 1^ 3 2]
-    + 0.08950028803070323 [0^ 2^ 0 2]
-    + 0.3353663891543792 [0^ 2^ 2 0]
-    + 0.08950028803070323 [0^ 3^ 1 2]
-    + 0.3353663891543792 [0^ 3^ 3 0]
-    + 0.3411947665760211 [1^ 0^ 0 1]
-    + 0.08950028803070323 [1^ 0^ 2 3]
-    + 0.3411947665760211 [1^ 1^ 1 1]
-    + 0.08950028803070323 [1^ 1^ 3 3]
-    + 0.08950028803070323 [1^ 2^ 0 3]
-    + 0.3353663891543792 [1^ 2^ 2 1]
-    + 0.08950028803070323 [1^ 3^ 1 3]
-    + 0.3353663891543792 [1^ 3^ 3 1]
-    + 0.3353663891543792 [2^ 0^ 0 2]
-    + 0.08950028803070323 [2^ 0^ 2 0]
-    + 0.3353663891543792 [2^ 1^ 1 2]
-    + 0.08950028803070323 [2^ 1^ 3 0]
-    + 0.08950028803070323 [2^ 2^ 0 0]
-    + 0.352552816086392 [2^ 2^ 2 2]
-    + 0.08950028803070323 [2^ 3^ 1 0]
-    + 0.352552816086392 [2^ 3^ 3 2]
-    + 0.3353663891543792 [3^ 0^ 0 3]
-    + 0.08950028803070323 [3^ 0^ 2 1]
-    + 0.3353663891543792 [3^ 1^ 1 3]
-    + 0.08950028803070323 [3^ 1^ 3 1]
-    + 0.08950028803070323 [3^ 2^ 0 1]
-    + 0.352552816086392 [3^ 2^ 2 3]
-    + 0.08950028803070323 [3^ 3^ 1 1]
-    + 0.352552816086392 [3^ 3^ 3 3]
+    0.3411947665 [0^ 0^ 0 0] +
```

Comment:
The function actually returns the new result, which is in agreement with the result returned by the differentiable solver.